### ISSUE TITLE
0.1.2 helm chart cannot update helm chart in-place

### ISSUE THREAD

**Issue by apogrebnyak** [jenkinsci/kubernetes-operator#349]
# Expected Behavior
Change some plugin version, and update helm chart normally

# Actual Behavior
Getting this error

```
Error: cannot patch "jenkins-backup" with kind PersistentVolumeClaim: PersistentVolumeClaim "jenkins-backup" is invalid: spec: Forbidden: is immutable after creation except resources.requests for bound claims
```

# Additional Info

- Kubernetes version:

  **Output of `kubectl version`:**

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-03-26T06:17:09Z", GoVersion:"go1.14", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.9-eks-502bfb", GitCommit:"502bfb383169b124d87848f89e17a04b9fc1f6f0", GitTreeState:"clean", BuildDate:"2020-02-07T01:31:02Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
```

- Jenkins Operator version:

```
0.4.0
```

**Comment by tomaszsek** [comment]
Can be fixed by https://github.com/helm/helm/issues/6642#issuecomment-542836520?

**Comment by apogrebnyak** [comment]
I am using helm through terraform provider.
Setting [force_update](https://www.terraform.io/docs/providers/helm/r/release.html#force_update) flag to `true` still resulted in the same error

**Comment by tomaszsek** [comment]
Fixed in `0.3.2` Helm chart
